cmd/kubeadm/app/cmd/alpha/selfhosting.go

```diff
@@ -47,46 +53,64 @@ var (
 		` + cmdutil.AlphaDisclaimer)
 
 	selfhostingExample = normalizer.Examples(`
-		# Converts a static Pod-hosted control plane into a self-hosted one, 
-		# functionally equivalent to what generated by kubeadm init executed
-		# with --feature-gates=SelfHosting=true. 
+		# Converts a static Pod-hosted control plane into a self-hosted one. 
 
 		kubeadm alpha phase selfhosting convert-from-staticpods
```

Comment:
please remove the empty line above this `57`.

Reply:
this would then be inconsistent with the ones around it, are you sure?
```var (
	selfhostingLongDesc = normalizer.LongDesc(`
		Converts static Pod files for control plane components into self-hosted DaemonSets configured via the Kubernetes API.

		See the documentation for self-hosting limitations.

		` + cmdutil.AlphaDisclaimer)

	selfhostingExample = normalizer.Examples(`
		# Converts a static Pod-hosted control plane into a self-hosted one. 

		kubeadm alpha phase self-hosting convert-from-staticpods
		`)
)```

Reply:
I'll do it. It's just how it was, and it matches the other cases above it. If you do really want me to change it do you want me to change the code above it too to match?

Reply:
in the new phase commands we tend to have:
```
# comment
example code
```